src/wallet/spend.cpp

```diff
@@ -672,10 +698,9 @@ bool CWallet::CreateTransactionInternal(
                 return false;
             }
 
-            // Get long term estimate
-            CCoinControl cc_temp;
-            cc_temp.m_confirm_target = chain().estimateMaxBlocks();
-            coin_selection_params.m_long_term_feerate = GetMinimumFeeRate(*this, cc_temp, nullptr);
+            // Get the long term fee estimate. This estimate is really the fee rate at which we are still willing to consolidate inputs.
+            // By default, we use 10 sat/vbyte.
+            coin_selection_params.m_long_term_feerate = CFeeRate(10, COIN); // Use 10 sat per vbyte
```

Comment:
What's the rationale behind `10 sat per vbyte`? Why that number?

Seems a bit scary to hard-code a value here.

Reply:
@Xekyo suggested it based on his analysis and experience.

This should really be a configurable option, I just didn't implement that yet.

Reply:
Forgot to mention that this was made a configurable option.